open-metadata-implementation/access-services/glossary-view/glossary-view-client/src/main/java/org/odpi/openmetadata/accessservices/glossaryview/client/GlossaryViewClient.java

```diff
@@ -0,0 +1,599 @@
+     * @throws GlossaryViewOmasException if a problem occurs on the omas backend
+     */
+    public GlossaryViewEntityDetailResponse getSubcategories(String serverName, String userId, String categoryGUID,
+                                                             Integer from, Integer size)
+            throws PropertyServerException, InvalidParameterException, GlossaryViewOmasException {
+
+        return getMultipleRelatedEntitiesPagedResponse("getSubcategories", GET_SUBCATEGORIES, serverName,
+                userId, categoryGUID, from, size);
+    }
+
+    /**
+     * Extract a category's external glossaries
+     *
+     * @param userId calling user
+     * @param serverName instance to call
+     * @param categoryGUID category GUID
+     *
+     * @return EntityDetailResponse external glossary links
+     *
+     * @throws PropertyServerException if a problem occurs while serving the request
+     * @throws InvalidParameterException if parameter validation fails
+     * @throws GlossaryViewOmasException if a problem occurs on the omas backend
+     */
+    public GlossaryViewEntityDetailResponse getExternalGlossariesOfCategory(String serverName, String userId, String categoryGUID,
+                                                                            Integer from, Integer size)
+            throws PropertyServerException, InvalidParameterException, GlossaryViewOmasException {
+
+        return getMultipleRelatedEntitiesPagedResponse("getExternalGlossariesOfCategory", GET_CATEGORY_EXTERNAL_GLOSSARIES,
+                serverName, userId, categoryGUID, from, size);
+    }
+
+    /**
+     * Extract a term definition
+     *
+     * @param userId calling user
+     * @param serverName instance to call
+     * @param termGUID term GUID
+     *
+     * @return EntityDetailResponse term
+     *
+     * @throws PropertyServerException if a problem occurs while serving the request
+     * @throws InvalidParameterException if parameter validation fails
+     * @throws GlossaryViewOmasException if a problem occurs on the omas backend
+     */
+    public GlossaryViewEntityDetailResponse getTerm(String serverName, String userId, String termGUID)
+            throws PropertyServerException, InvalidParameterException, GlossaryViewOmasException {
+
+        return getSingleEntityResponse("getTerm", GET_TERM, serverName, userId, termGUID);
+    }
+
+    /**
+     * Extract terms within a glossary
+     *
+     * @param userId calling user
+     * @param serverName instance to call
+     * @param glossaryGUID glossary GUID
+     *
+     * @return EntityDetailResponse terms
+     *
+     * @throws PropertyServerException if a problem occurs while serving the request
+     * @throws InvalidParameterException if parameter validation fails
+     * @throws GlossaryViewOmasException if a problem occurs on the omas backend
+     */
+    public GlossaryViewEntityDetailResponse getTermsOfGlossary(String serverName, String userId, String glossaryGUID,
```

Comment:
The result of the client calls should be a standard Java interface with the beans returned and the exceptions called out.  The response object contains the exception information which should be returned as exceptions

Reply:
Changed return types from wrapper to specific pojos